docs/resources/database.md

```diff
@@ -10,7 +10,18 @@ description: |-
 
 A Tabular Database
 
+## Example Usage
 
+```terraform
+data "tabular_warehouse" "warehouse" {
+  name = "funhouse"
+}
+
+resource "tabular_database" "database" {
+  warehouse_id = data.tabular_warehouse.warehouse.id
+  name         = "other!"
```

Comment:
maybe we want to avoid actively encouraging weird chars in db names at the very least?  not sure if some engines will barf on that
```suggestion
  name         = "other"
```